### DISCUSSION TITLE
Performance voters

### DISCUSSION THREAD

**Discussion by CarlosHSF99** [cesium/karaokium#10] [Ideas]
For every performance, clicking the number of votes would show a list of the voters.
It could also show their associated reactions (assuming #6 is implemented with the current most voted option).

**Comment by nelsonmestevao** [comment]
Should these be accessible to the participants or only the admins? Shouldn't the votes be anonymous? I think the reactions would be cool to be able to see, though.

**Reply by CarlosHSF99** [reply]
I was thinking it would be interesting for the admins to know who voted and for the participants to see who reacted what.
But there's probably better places and ways to show the reactions, like how discord does it:
![image](https://user-images.githubusercontent.com/44176623/166160301-f611b14f-67b8-4093-bf0e-46bc70ea538f.png)

So, it would be only accessible to the admins to know who participated, but the value of the vote would still be anonymous.

**Reply by MFJess** [reply]
I agree that it's probably better for everyone if the votes were anonymous.
